apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/ShardingSettings.java

```diff
@@ -35,6 +50,24 @@ public ShardingSettings(boolean enabled, String digestAlgorithm) {
         this(enabled, DEFAULT_SHARDING_FREQUENCY, digestAlgorithm);
     }
 
+    /**
+     * Used for unit test and by BlockchainConfigUpdater
+     * @param startHeight configured height
+     * @param shardingSettings corresponding shard settings
+     */
+    public ShardingSettings(int startHeight, ShardingSettings shardingSettings) {
```

Comment:
It is easy to create same method that create ShardSettings for unit test directly inside unit test , not inside testable code.

Reply:
That was removed